### DISCUSSION TITLE
挂载迅雷网盘和联通网盘

### DISCUSSION THREAD

**Discussion by ningfeiyu** [AlistGo/alist#5876] [Q&A]
按照文档和页面挂载很顺利，迅雷上的是直接不能播发，联通上的是能播放，但是频繁的验证码多次，然后就需要重新填写token。大家是怎么解决的呢？

**Comment by ningfeiyu** [comment]
联通报错failed link: failed get link: request failed with rsp_code: 8005,rep_desc: 登录失败！

**Comment by ningfeiyu** [comment]
联通网盘有手机版和ipad，是不是可以加UA进来就不会刷新验证码了？、

**Comment by doKill** [comment]
同，楼主解决了吗

**Comment by Simonzhang8** [comment]
迅雷因为是本机代理/代理地址，所以需要依赖服务器/中转机器的带宽，如果带宽低的就没办法放视频

**Reply by 1109wuwuwu** [reply]
我看迅雷还有，迅雷浏览器，迅雷专业版，我想问一下我看他们都有302，我有svip能达到客户端的速度吗？

**Comment by lylcc520** [comment]
最近挂载的迅雷频繁掉线  每次nas重启后  就要重新获取一次token   我已经搞了好几天  看了alist官方教程和哔哩哔哩 都没有解决

**Comment by lylcc520** [comment]
302 在建立流后就是安装终端（理论上是能做到满速的）
注意这里终端是你安装 CD2 的终端


CD2 不知道挂载的机制是啥，我 nas 安装的 docker CD2 挂载迅雷一两比断开报错
&nbsp;我的迅雷和 CD 都是会员







------------------ 原始邮件 ------------------
发件人: Yehao Tan ***@***.***&gt;
发送时间: 2024年10月30日 14:51
收件人: AlistGo/alist ***@***.***&gt;
抄送: lylcc520 ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [AlistGo/alist] 挂载迅雷网盘和联通网盘 (Discussion #5876)





 
我看迅雷还有，迅雷浏览器，迅雷专业版，我想问一下我看他们都有302，我有svip能达到客户端的速度吗？

**Comment by lylcc520** [comment]
但是我的 alist 目前挂载迅雷云盘只能本地 &nbsp;专业版太复杂了
而且现在 alist 挂载迅雷 只要 alist 容器重启 也会断和 CD2
不知道是不是迅雷那边的问题还是哪里出问题，一起的老版本没有过


希望 alist 能找到原因





------------------ 原始邮件 ------------------
发件人: Yehao Tan ***@***.***&gt;
发送时间: 2024年10月30日 14:51
收件人: AlistGo/alist ***@***.***&gt;
抄送: lylcc520 ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [AlistGo/alist] 挂载迅雷网盘和联通网盘 (Discussion #5876)





 
我看迅雷还有，迅雷浏览器，迅雷专业版，我想问一下我看他们都有302，我有svip能达到客户端的速度吗？
